packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -59,6 +59,9 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
   /// Returns the currentAtSign associated with the SyncService
   String get currentAtSign => _atClient.getCurrentAtSign()!;
 
+  /// A local AtKey to persist the last received server commitId
+  late AtKey _lastReceivedServerCommitIdAtKey;
```

Comment:
Please mark this variable as final as well as late

Reply:
Sure Gary.